tests/unit/Storefront/Page/Robots/Parser/ParsedRobotsTest.php

```diff
@@ -0,0 +1,264 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Storefront\Page\Robots\Parser;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Storefront\Page\Robots\Parser\ParsedRobots;
+use Shopware\Storefront\Page\Robots\Parser\ParseIssue;
+use Shopware\Storefront\Page\Robots\Parser\ParseIssueSeverity;
+use Shopware\Storefront\Page\Robots\Struct\RobotsDirective;
+use Shopware\Storefront\Page\Robots\Struct\RobotsDirectiveType;
+use Shopware\Storefront\Page\Robots\Struct\RobotsUserAgentBlock;
+
+/**
+ * @internal
+ */
+#[CoversClass(ParsedRobots::class)]
+class ParsedRobotsTest extends TestCase
```

Comment:
please remove the whole tests and then check again, that you only test real logic from that class. there is no value in testing getters and setter in 1000 different ways...